roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    node_selector: "{{ openshift_prometheus_node_selector | replace(':','=') | replace('{', '') | replace('}', '') | replace(' ', '') | replace(\"u\'\", '') | replace(\"\'\", '') }}"
```

Comment:
@zgalor i was suggesting putting this kind of function into a role-specific filter plugin. An example of how we've done it in the past here: https://github.com/openshift/openshift-tools/blob/prod/ansible/roles/openshift_aws_persistent_volumes/filter_plugins/filters.py